src/main/java/org/gridsuite/modification/server/modifications/ModificationUtils.java

```diff
@@ -1027,5 +1030,48 @@ public boolean isValidFilter(Reporter subReporter,
 
         return true;
     }
+
+    public static List<IdentifiableAttributes> getIdentifiableAttributes(Map<UUID, FilterEquipments> exportFilters, Map<UUID, FilterEquipments> filtersWithWrongEquipmentIds, List<FilterInfos> filterInfos, Reporter subReporter) {
+        filterInfos.stream()
+                .filter(f -> !exportFilters.containsKey(f.getId()))
+                .forEach(f -> createReport(subReporter,
+                        "filterNotFound",
+                        String.format("Cannot find the following filter: %s", f.getName()),
+                        TypedValue.WARN_SEVERITY));
+
+        return filterInfos
+                .stream()
+                .filter(f -> !filtersWithWrongEquipmentIds.containsKey(f.getId()) && exportFilters.containsKey(f.getId()))
+                .flatMap(f -> exportFilters.get(f.getId())
+                        .getIdentifiableAttributes()
+                        .stream())
+                .filter(distinctByKey(IdentifiableAttributes::getId))
```

Comment:
No need to log the duplicated ids ?

Reply:
@FranckLecuyer @YenguiSeddik I removed this part, it will be handled in a separate PR (there is already a story created) as there are some questions about logging and I found some issues with the logging of scaling features. 
I reviewed and treated all your other comments.